front/app/containers/Admin/tools/messages.ts

```diff
@@ -0,0 +1,72 @@
+import { defineMessages } from 'react-intl';
+
+export default defineMessages({
+  tools: {
+    id: 'app.containers.Admin.tools',
```

Comment:
To make it easier to find, we can add an extra part to the id. This id is shared between all other strings.